docs/TRANSLATING.md

```diff
@@ -1,80 +1,289 @@
+sequence_ `\"`.
 
-## String-specific tips
+Most dedicated `PO` editors offer escape sequence syntax highlighting.
 
-### Format strings
+### C-style format strings
 
-Many strings contain strangely looking sequences starting from the percentage sign,
-like `%s` or `%02d`. These are so-called _printf format strings_, used by C/C++
-programming languages; they are replaced with another string or number, using the
-specified format.
+Many strings contain strange-looking sequences starting from the percentage
+sign, like `%s` or `%02d`. These are so-called _printf format strings_, used by
+C/C++ programming languages; they are replaced with another string or number,
+using the specified format.
 
 You might need to understand the notation to properly format the output (like tables,
-when the translated headers have different length than in English). Search the web
-to learn more about it, here are a few links:
+when the translated headers have different lengths than in English). Search the
+web to learn more about it; here are a few links:
 
 - [printf article on Wikipedia](https://en.wikipedia.org/wiki/Printf)
 - [printf article on cplusplus.com](https://cplusplus.com/reference/cstdio/printf)
 - [printf article on cppreference.com](https://en.cppreference.com/w/cpp/io/c/fprintf)
 
+The `c-format` flag notes that the message can contain a C-style format string.
+Some dedicated `PO` editors might offer format string syntax highlighting or
+even error checking.
+
+If DOSBox Staging detects a mismatched (incompatible) format string, it refuses
+to use the translation for this particular message, prints out a warning log,
+and marks the message as `fuzzy` the next time the `PO` file is saved.
+
+## Steps before creating a pull request
+
+There are still a couple of things to do after you are done with editing the
+`PO` file.
+
+UTF-8 can store the same exact text in several different ways. There are some
+standard ones, however, called _normalized forms_. Although
+DOSBox Staging should be able to handle non-normalized translation files, and
+normally text editors save normalized files already, it is still highly
+recommended to normalize the updated `PO` file.
+
+In the `extras/translations` directory, you'll find two scripts: `normalize.sh`
+(for Linux and macOS) and `normalize.bat` (for Windows). The scripts convert
+the translation files in the `resources/translations` directory to the
+NFC-normalized form (recommended by DOSBox Staging and used by default by most
+UTF-8 capable text editors), with Unix line endings. 
+
+Now you need to clean up the file. Once again start DOSBox Staging (this time
+with your updated `PO` file) and once again execute the commands:
+
+```bat
+config -set language=xx
+config -wl xx.po
+```
+
+This will write the `PO` file again, using the standard DOSBox Staging
+formatting style. It is especially important if you used a dedicated `PO` file
+editor.
+
+> **Note**
```

Comment:
`> [!IMPORTANT]` would make this stand out more.

See:
https://blog.solichain.com/how-to-create-admonitions-in-github-markdown-ab580debcfdc